### DISCUSSION TITLE
[OP Stack Deployment] How can we use Optimism bridge to move Ocean tokens from Ethereum mainnet to OP mainnet?

### DISCUSSION THREAD

**Discussion by LoznianuAnamaria** [ethereum-optimism/developers#52] [Deploying the OP Stack] [answered]
### Issue Description

We are currently working to deploy the Ocean stack on Optimism. For this, we need to be able to bridge Ocean tokens to the OP mainnet. How can we add [OCEAN](https://etherscan.io/token/0x967da4048cD07aB37855c090aAF366e4ce1b9F48) to your [bridge](https://app.optimism.io/bridge/deposit)?

PS: On the same note, we need to first deploy the contracts on your OP Sepolia testnet. How can we get more test tokens? At the current rate of your [faucet](https://faucet.quicknode.com/optimism/sepolia) - 0.05, it would take us too many days to get the required amount 👴 
Thank you very much 🙏 

### Additional Information

This is part of a grant that was given to Ocean Protocol by the Optimism Foundation Grant.

**Comment by opfocus** [comment, marked as answer]
bridge Ocean tokens to the OP mainnet：  
- First, you need to create a mapped token on the OP-Mainnet, you can refer to https://github.com/ethereum-optimism/optimism-tutorial/tree/main/standard-bridge-standard-token. 
- Once created, you can proceed with the tutorial or add it on the bridgeUI of OP-Mainnet, then bridge via https://app.optimism.io/bridge/deposit.
-  If you are performing similar operations in Sepolia, you'll need to use the SDK, referring to SDK support for Sepolia: https://github.com/ethereum-optimism/optimism/blob/develop/packages/sdk/src/interfaces/types.ts. 
Regarding the issue of test tokens, I don't have enough test tokens to share. @sbvegan

**Reply by sbvegan** [reply]
Hello @AnaLoznianu, please go ahead and send me your test wallet address. I can send you some OP Sepolia testnet ether. In addition to what @opfocus said, please make sure to test transfers in both directions (L1 --> L2 && L2 --> L1). Our standard bridge only supports standard ERC20 tokens. Once you're confident on the testnet bridging, you can follow this [guide](https://static.optimism.io/) to get added to the bridge UI.

**Reply by LoznianuAnamaria** [reply]
Great, thank you very much! Our engineer will follow the resources you shared and we'll get back if we have any questions. This is our address: 0xC7EC1970B09224B317c52d92f37F5e1E4fF6B687

**Reply by sbvegan** [reply]
@AnaLoznianu okay sounds good! I've sent 1 testnet ether on Sepolia and OP Sepolia. When you get a chance, can you please mark this thread as answered and open a new one if y'all have additional questions?

**Reply by alexcos20** [reply]
Thanks, I got the test ETH.   Will start deploying the contract and open a PR for bridge.

**Reply by alexcos20** [reply]
PR Ready:  https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/573

**Reply by sbvegan** [reply]
@alexcos20 one request from our end: https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/573#issuecomment-1808394005
